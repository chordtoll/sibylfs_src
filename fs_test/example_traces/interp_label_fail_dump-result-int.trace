@type script
dump-result /
end dump-result
