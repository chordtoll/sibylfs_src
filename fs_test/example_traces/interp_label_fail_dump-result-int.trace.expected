# processing file 'example_traces/interp_label_fail_dump-result-int.trace' ...
# 
# Fatal error: File "example_traces/interp_label_fail_dump-result-int.trace", line 2, characters 0-14:
# dump-result not allowed in script
# processing file 'example_traces/interp_label_fail_dump-result-int.trace' failed
